cypress/e2e/settings.cy.js

```diff
@@ -1,32 +1,54 @@
 /// <reference types='cypress' />
 /// <reference types='../support' />
 
+import SettingsPageObject from '../support/pages/settings.pageObject';
+
 describe('Settings page', () => {
-  before(() => {
+  const settingsPage = new SettingsPageObject();
 
+  before(() => {
+    cy.task('db:clear');
   });
 
   beforeEach(() => {
+const settingsPage = new SettingsPageObject();
```

Comment:
The `settingsPage` object is being re-declared inside `beforeEach`, which is unnecessary since it is already declared at the top of the `describe` block. Consider removing this line to avoid redundancy.